docs/cron/export.sh

```diff
@@ -4,10 +4,10 @@ set -e
 PATH=$PATH:/usr/local/mysql/bin
 ROOT='/var/www-prod'
 
-mysql -u "$DB_USER" -p"$DB_PASS" "$DB" < /var/www-prod/docs/database/scripts/weekly_exports.sql
+mysql -u "$DB_USER" -p"$DB_PASS" "$DB" < $ROOT/docs/database/scripts/weekly_exports.sql
 
 DL_DIR="/var/www-downloads/exports"
```

Comment:
If you make this a variable in `app_local.php`, you should also remove this line and edit the cron job runner `docs/cron/runner.sh` so that it passes the variable to this script instead.